content/en/docs/ops/configuration/security/security-policy-patterns/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Security Policy Patterns
```

Comment:
Security policy examples? 

Do you want to show some examples for authentication policies as well? All examples here are authorization policies. Maybe some examples of how authentication policy and authorization policy can be used together.

Reply:
updated the title, I want to add some examples for authn as well but just doesn't have good examples at hand.

Let me know if you have suggestions or we can add more examples later.